backend/seed_data/test_cases.yaml

```
@@ -116,3 +116,33 @@ test_cases:
       - input: "[2,3,0,0,4,2]"
         outputs:
           - "[0,0,0,0,0,0]"
+
+# Graphs
```

YAMLlint (1.35.1)

[warning] comment not indented like content

(comments-indentation)